src/commands/force/org/delete.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as os from 'os';
+import { flags, FlagsConfig, SfdxCommand } from '@salesforce/command';
+import { Messages, Org } from '@salesforce/core';
+import { SandboxOrgConfig } from '@salesforce/core/lib/config/sandboxOrgConfig';
```

Comment:
`SandboxOrgConfig` doesn't seem to be intentionally exposed.

Why not wrap all of `(await this.org.getSandboxOrgConfigField(SandboxOrgConfig.Fields.PROD_ORG_USERNAME))`;
in a new `org.isSandbox()` method?